Fix Django3 deprecation warnings

zerolab opened the pull request:
Fixes #94

* Drops Django 1.11
* Set minimum Wagtail version to 2.7 (current LTS)
* Adds Python 3.8 and Django 3.0 to tox environments

thenewguy reviewed (approved):
Are we dropping support for py37? If not, maybe run LTS tests and latests under py36 and py37 in addition to py38.

zerolab commented:
@thenewguy sharp-eyed as ever! updated to run the LTS tests with all of its supported python versions. + fixed some references in tox (wag210 instead of wag10 (or wag2.10))
